release/openapi/swagger/src/api.swagger.json

```diff
@@ -9,12 +9,17 @@
     "https"
   ],
   "swagger": "2.0",
+  "externalDocs": {
+    "description": "Join our Discord",
+    "url": "https://discord.com/invite/yVhQ7rC"
+  },
```

Comment:
This file is generated - any changes to it will be overwritten with a `make build` and the CI checks that `make build` is a no-op.